test/harry/main/org/apache/cassandra/harry/model/PartitionState.java

```diff
@@ -223,7 +232,7 @@ private RowState updateRowState(RowState currentState, IntFunction<Bijections.Bi
 
                 assert lts >= currentState.lts[i] : String.format("Out-of-order LTS: %d. Max seen: %s", lts, currentState.lts[i]); // sanity check; we're iterating in lts order
 
-                if (currentState.lts[i] == lts)
+                if (lts != MagicConstants.NO_TIMESTAMP && currentState.lts[i] == lts)
```

Comment:
right now my test doesn't own the timestamps, so we disable.  When you disable then we end up trying to compare objects rather than replacing with the latest... needed this step to avoid such a problem